src/libsync/networkjobs.cpp

```diff
@@ -589,6 +590,42 @@ bool PropfindJob::finished()
 
 /*********************************************************************************************/
 
+AvatarJob::AvatarJob(AccountPtr account, QObject *parent)
+    : AbstractNetworkJob(account, QString(), parent)
+{
+    _avatarUrl = Utility::concatUrlPath(account->url(), QString("remote.php/dav/avatars/%1/128.png").arg(account->davUser()));
```

Comment:
Does not work for me with this URL. (owncloud master)
The URL that works for me is `index.php/avatar/%1/128`


@DeepDiver1975  What's the exact URL of the  avatars?